### PR TITLE
Update AFDX sample - Front Door Standard/Premium (Preview) with Azure Storage static website origin

### PR DESCRIPTION
**Pull request by johndowns**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Update dependencies to conform to new Front Door ARM API requirements.
* Add static website content pages to make this a nicer experience to deploy and test.

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

File: 201-front-door-standard-premium-storage-static-website/modules/storage-static-website.bicep [hunk 1]
```diff
@@ -38,7 +39,7 @@ resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-
   location: location
 }
 
-resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
+resource roleAssignmentContributor 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
   scope: storageAccount
   name: guid(resourceGroup().id, managedIdentityName, storageAccountContributorRoleDefinitionId)
```
Comment (bmoore-msft) [inline 201-front-door-standard-premium-storage-static-website/modules/storage-static-website.bicep:44]: still getting used to bicep - but I think this is going to be common in your samples?

For the roleAssignment you want to use the resourceId of the MSI in the name, not just the name of the resource.

Can you take a pass through the other samples and check those as well?

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 201-front-door-standard-premium-storage-static-website/modules/storage-static-website.bicep (hunk 1 above)
Comment (johndowns) [reply 201-front-door-standard-premium-storage-static-website/modules/storage-static-website.bicep]: Fixed. I will check the other samples and do PRs if needed. This is a common pattern I'm following but only where deployment scripts are necessary. If we can fix up the RPs so the scripts aren't necessary then I can probably remove them :)

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Pull request merged**